### PR TITLE
Change font to improve font performance and LH score

### PR DESCRIPTION
**Pull request by igorbrasileiro**
## What's the purpose of this pull request?
Change font-display to fallback, and subset fonts to Unicode range from used characters using glyphhanger

## How does it work?
Change font face definitions to improve performance, recreated font files with subset glyphs using `glyphhanger`

Results after glyphhanger (used PDP link, but home can be used too. PLP isn't recommended because has less characters)
image

## How to test it?
- LHCI already does it

## References
https://web.dev/reduce-webfont-size/#unicode-range-subsetting
https://web.dev/fast/#otimize-webfonts
https://web.dev/optimize-webfont-loading/
https://web.dev/font-best-practices/

## Checklist
- [] CHANGELOG entry added

### CODE REVIEW

File: src/styles/fonts.css [hunk 1]
```diff
@@ -33,12 +33,12 @@
   font-weight: 900;
   font-family: "Lato";
   font-style: normal;
-  font-display: swap;
+  font-display: fallback;
   src: url("../fonts/lato-v20-latin-900.eot"); /* IE9 Compat Modes */
   src: local(""),
     url("../fonts/lato-v20-latin-900.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/lato-v20-latin-900.woff2") format("woff2"),
```
Comment (tlgimenes) [inline src/styles/fonts.css:40]: please, make sure to remove these from the repo

**Review by tlgimenes — APPROVED** [review]
nice! great improvement!

File: src/styles/fonts.css (hunk 1 above)
Comment (igorbrasileiro) [reply src/styles/fonts.css:40]: I left here, if needed in future, so we can regenerate the new font subset

**Comment by icazevedo** [comment]
why did it drop 20 points on LHCI? is it a one-time thing?

**Comment by filipewl** [comment]
Nice!

I hadn't noticed [the Perf drop for Home](http://lighthouse-ci.staging-2b.ingress.vtex.io/app/projects/base.store/compare/d2027feb-143f-4114-8694-4ce82d65ad1c?compareUrl=%2F) till @icazevedo pointed it out. Now, why hasn't VTEX Sites' Lighthouse check failed due to the `categories:performance` 95 threshold? 🤔 

Maybe it's a one-time thing, so shall we retry?

**Review by eduardoformiga — APPROVED** [review]
Good Job! 👏

**Review by filipewl — APPROVED** [review]
Very interesting optimization, I wasn't aware of that. 
LGTM once we confirm LH score for Home is okay!

Visual changes seem very minimal (note the "J" in "Just Arrived"):
CleanShot 2022-02-16 at 14 58 20

**Comment by igorbrasileiro** [comment]
@filipewl fixed

**Pull request merged**